setup.py

```diff
@@ -1174,5 +1174,6 @@ def run(self):
         RUAMEL_YAML_VERSION,
         "lru-dict==1.1.8",
         MARKO_VERSION,
+        "py_arkworks_bls12381==0.3.2",
```

Comment:
For reference: 0.3.2 is the first version which also includes multithreading, so you may see a difference in speed with algorithms that can be multithreaded like multiexp_unchecked